src/sentry/static/sentry/app/views/onboarding/configure.jsx

```diff
@@ -88,7 +88,11 @@ const Configure = createReactClass({
           <h2 style={{marginBottom: 30}}>
             {t('Configure your application')}
             {!hasSentRealEvent && (
-              <CreateSampleEvent params={this.props.params} source="header" />
+              <div className="pull-right">
+                <CreateSampleEvent project={this.project} source="header">
```

Comment:
Can we rename this component to `CreateSameEventButton`?

Reply:
👍